components/activities/poll-banner/poll-banner.tsx

```diff
@@ -0,0 +1,23 @@
+import { React, useStyleSheet } from "../../../deps.ts";
+import { Poll } from "../../../types/mod.ts";
+import { isPrediction } from "../../../shared/mod.ts";
+import { ActivityBannerProps } from "../activity-banner.tsx";
+import PredictionBanner from "./prediction-banner.tsx";
+import BasePollBanner from './base-poll-banner.tsx'
+
+import stylesheet from "./poll-banner.scss.js";
+
+const PollBanner = (
+  { activity }: ActivityBannerProps<Poll>,
+) => {
+  useStyleSheet(stylesheet);
+  return (
+    <>
+      {
+        isPrediction(activity) ? <PredictionBanner poll={activity} /> : <BasePollBanner poll={activity} />
```

Comment:
Nitpick but does this need to be in a fragment? Or can you return line 17 directly

Reply:
doesn't need to be a fragment, I can rm